From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Samuel Maddock <smaddock@slack-corp.com>
Date: Fri, 28 Mar 2025 20:22:26 -0400
Subject: revert: Enable CREL for arm32 targets

Enabling CREL on Linux ARM64 seems to cause it to segfault. Disable for Electron
as its one of our supported platforms.
https://chromium-review.googlesource.com/q/I3a62f02f564f07be63173b0773b4ecaffbe939b9

diff --git a/build/config/compiler/BUILD.gn b/build/config/compiler/BUILD.gn
index b0b194e215ff7cf60a58f87fcbd157d32e9de13f..2afd69b60e8874042f98d64830d3cd77d2211e7a 100644
--- a/build/config/compiler/BUILD.gn
+++ b/build/config/compiler/BUILD.gn
@@ -617,7 +617,8 @@ config("compiler") {
 
     # Enable ELF CREL (see crbug.com/357878242) for all platforms that use ELF
     # (excluding toolchains that use an older version of LLVM).
-    if (is_linux && !llvm_android_mainline &&
+    # TODO(crbug.com/376278218): This causes segfault on Linux ARM builds.
+    if (is_linux && !llvm_android_mainline && current_cpu != "arm" &&
         default_toolchain != "//build/toolchain/cros:target") {
       cflags += [ "-Wa,--crel,--allow-experimental-crel" ]
     }
